Makefile

```diff
@@ -0,0 +1,60 @@
+# SPDX-FileCopyrightText: 2022 Serokell <https://serokell.io>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+.PHONY: xrefcheck test lint clean bats all
+
+# Build target from the common utility Makefile
+MAKEU = $(MAKE) -C make/
+
+MAKE_PACKAGE = $(MAKEU) PACKAGE=xrefcheck
+
+xrefcheck:
+	$(MAKE_PACKAGE) dev
+
+# Runs all tests.
+test:
```

Comment:
Can you add some usage examples here, like the ones we have in coffer?

(Except we should replace `--pattern` with `--match`, because we're using `hspec` instead of `tasty`)